modules/code-editor/src/views/full/SidePanel.tsx

```diff
@@ -32,15 +34,16 @@ export default class PanelContent extends React.Component<Props> {
       return
     }
 
-    const { actionsBot, actionsGlobal, hooksGlobal } = this.props.files
+    const { actionsBot, actionsGlobal, hooksGlobal, botConfigs } = this.props.files
 
     const actionFiles = []
     actionsBot && actionFiles.push({ label: `Bot (${window['BOT_NAME']})`, files: actionsBot })
     actionsGlobal && actionFiles.push({ label: 'Global', files: actionsGlobal })
 
     const hookFiles = [hooksGlobal && { label: 'Global', files: hooksGlobal }]
+    const botConfigFiles = [botConfigs && { label: 'All Bots', files: botConfigs }]
 
-    this.setState({ actionFiles, hookFiles })
+    this.setState({ actionFiles, hookFiles, botConfigs: botConfigFiles })
```

Comment:
Do I understand that you load configs for all bots ? If yes I think you should fetch only the current bot config

Reply:
Yep, it loads for all bots. I wasn't sure on how we want to handle that. Maybe like actions, display the current bot config, and if global is enabled, show all of the bot config files?